internal/common/function/function_test.go

```diff
@@ -55,9 +55,14 @@ const (
 	testNonAdminRestoreUUID       = "12345678-1234-1234-1234-123456789abc"
 	testNonAdminRestoreNamespace  = "non-admin-restore-namespace"
 	testNonAdminSecondRestoreName = "non-admin-second-restore-name"
-	defaultStr                    = "default"
 	invalidInputEmptyNamespace    = "Invalid input - empty namespace"
-	invalidInputEmptyNsErr        = "invalid input: namespace, labelKey, and labelValue must not be empty"
+	defaultStr                    = "default"
```

Comment:
maybe drop the `Str` prefix from var names

Reply:
@shubham-pampattiwar in go "default" is reserved keyword / because it's used in switch statements, so what else should I change this to?

Reply:
From checking code, this is only used to convey the default ns name.

I suggests `namespaceDefault` to convey its utility.

Reply:
`defaultNS` or `defaultNamespace` also works.

Reply:
changed.